client/src/components/Player.ts

```diff
@@ -54,10 +78,23 @@ export class Player extends EnginePlayer {
     }
 
     super.update(dt);
+
+    this.spritesContainer.children.forEach(sprite => {
```

Comment:
I think its enough to just update spriteContainer with x, y.

If you update its children, their position will be relativetothe parent

Reply:
true